package.json

```diff
@@ -335,7 +335,7 @@
 		"test:unit:profile": "wp-scripts --cpu-prof test-unit-js --runInBand --no-cache --verbose --config test/unit/jest.config.js ",
 		"test:unit:php:setup": "wp-env start",
 		"test:unit:php:setup:debug": "wp-env start --xdebug",
-		"test:unit:php:base": "wp-env run --env-cwd='wp-content/plugins/gutenberg' tests-wordpress vendor/bin/phpunit -c phpunit.xml.dist --verbose",
+		"test:unit:php:base": "wp-env run --env-cwd='wp-content/plugins/gutenberg' tests-wordpress vendor/bin/phpunit -c phpunit.xml.dist --verbose --filter WP_Theme_JSON_Gutenberg_Test",
```

Comment:
@anton-vlasenko We weren't able to run the PHP unit tests with the `--filter` option. We tried:

```
npm run test:unit:php -- --filter WP_Theme_JSON_Gutenberg_Test
```

...but it says "Invalid option: filter".

Any ideas why this might be happening?

Reply:
This is passing the `--filter` to `npm-run-all` instead of `phpunit`.

I would expect this to work:

```sh
npm run wp-env start
npm run test:unit:php:base -- --filter WP_Theme_JSON_Gutenberg_Test
```

Reply:
This is correct @ajlende